diff --git a/setup.py b/setup.py
index e06cf1c..cf55580 100644
--- a/setup.py
+++ b/setup.py
@@ -14,18 +14,7 @@ https://github.com/MDAnalysis/mdanalysis/tree/develop/package/setup.py
 retrieved 06/15/15
 """
 def detect_openmp():
-	"""Does this compiler support OpenMP parallelization?"""
-	compiler = new_compiler()
-	print("Checking for OpenMP support... ")
-	hasopenmp = hasfunction(compiler, 'omp_get_num_threads()')
-	needs_gomp = hasopenmp
-	if not hasopenmp:
-		compiler.add_library('gomp')
-	hasopenmp = hasfunction(compiler, 'omp_get_num_threads()')
-	needs_gomp = hasopenmp
-	if hasopenmp: print("Compiler supports OpenMP")
-	else: print( "Did not detect OpenMP support.")
-	return hasopenmp, needs_gomp
+	return False, False

 def hasfunction(cc, funcname, include=None, extra_postargs=None):
 	# From http://stackoverflow.com/questions/
